lib/Dialect/Comb/CombOps.cpp

```diff
@@ -282,6 +282,13 @@ LogicalResult ExtractOp::verify() {
   return success();
 }
 
+LogicalResult TruthTableOp::verify() {
+  ArrayAttr table = getLookupTable();
+  if (table.size() != (1ull << getInputs().size()))
```

Comment:
could you add a check that  `1ull << getInputs().size()` fits in 64 bit?

Reply:
Actually, we should technically check that it fits into a `size_t`...